docs/action_api.md

```diff
@@ -1,41 +1,42 @@
 
-By implementing the Action API your server becomes an "Action Hub" that users can connect Looker to. An Action Hub can expose one or more actions.
+By implementing the Action API your server becomes an "Action Hub" that can be connected to one or more Looker instances. An Action Hub can expose one or more actions.
 
 ## Endpoints
 
-In order to implement the Action API your server will need to expose a few endpoints for Looker to connect to:
+In order to implement the Action API your server will need to expose a few endpoints for Looker to call:
 
 - [Actions List Endpoint](#actions-list-endpoint)
 
   This endpoint lists all the actions your hub exposes. Each listed action contains metadata instructing Looker how to execute your action. It indicates what data payload formats are supported, what information (if any) to collect from the Looker administrator (such as API keys), and if the action also has an end-user facing Action Form.
 
-  The URL of this endpoint is entered into the Looker admin panel (as the "Action Hub URL") and is how users connect to your action hub.
+  The URL of this endpoint is entered into the Looker admin panel (as the "Action Hub URL") and is how admins connect to your action hub.
 
 - [Action Execute Endpoint](#action-execute-endpoint)
 
-  This is the endpoint that Looker will send the payload, form information, and other metadata in order to execute a given action. This is the main implementation of your action. This endpoint works the same way as a Looker Webhook endpoint.
+  This is the endpoint to which Looker will send the data payload, form information, and other metadata in order to execute a given action. This is the main implementation of your action. This endpoint works the same way as a Looker webhook endpoint.
 
 - (optional) [Action Form Endpoint](#action-form-endpoint)
 
-  If the action's definition specifies a form, Looker will ask this endpoint for a form template that should be displayed to the end user to configure the action. Since Looker will ask this endpoint for the form every time it's displayed, it's possible to dynamically change the form based on whatever information you like. For example, in a chat application the form might dynamically list all the available chat channels that data can be sent to.
+  If the action's definition specifies a form, Looker will ask this endpoint for a form template that will be displayed to the end user to configure the action (e.g. when using the action as a destination in the Schedule dialog). Since Looker will query this endpoint every time the form is displayed, it's possible to dynamically populate the form based on whatever information you like. For example, for a chat application the form might have a dropdown that dynamically list all the chat channels available at that moment. Or for an action that uses OAuth with the Google Analytics API, the form could dynamically list the web properties to which the current user has access.
```

Comment:
```suggestion
  If the action's definition specifies a form, Looker asks this endpoint for a form template to display to the user to configure the action (e.g. when using the action as a destination in the Schedule dialog). Since Looker will query this endpoint every time the form is displayed, it's possible to dynamically populate the form with custom user input fields. For example, for a chat application the form might have a drop-down that dynamically lists all the chat channels available at that moment. Or for an action that uses OAuth with the Google Analytics API, the form could dynamically list the web properties to which the current user has access.
```